packages/core/src/sass/account-wizard.scss

```diff
@@ -303,6 +306,16 @@
                 width: 78%;
                 text-align: center;
             }
+            p:first-child {
+                margin-top: 6rem;
+                background-color: var(--general-section-1);
+                font-size: 1.2rem;
+                height: 58px;
```

Comment:
```suggestion
                height: 5.8rem;
```

Reply:
We need exact height in pixels here as it's dependent on another element with this height.